doc/phpactor.txt

```diff
@@ -211,11 +255,11 @@ are available for you to copy:
     au FileType php,cucumber nmap <buffer> <Leader>o
         \ :PhpactorGotoDefinition<CR>
     au FileType php,cucumber nmap <buffer> <Leader>Oh
-        \ :PhpactorGotoDefinitionHsplit<CR>
+        \ :PhpactorGotoDefinition split<CR>
     au FileType php,cucumber nmap <buffer> <Leader>Ov
-        \ :PhpactorGotoDefinitionVsplit<CR>
+        \ :PhpactorGotoDefinition vsplit<CR>
     au FileType php,cucumber nmap <buffer> <Leader>Ot
-        \ :PhpactorGotoDefinitionTab<CR>
+        \ :PhpactorGotoDefinition tabnew<CR>
```

Comment:
Could possibly remove these mappings?

Reply:
I guess, it was there so I just updated it but we can remove it since there is another example already.

Reply:
Done